packages/compiler-cli/test/compliance/test_cases/r3_view_compiler_deferred/deferred_with_triggers_template.js

```diff
@@ -0,0 +1,17 @@
+function MyApp_Template(rf, ctx) {
+  if (rf & 1) {
+    $r3$.ɵɵtemplate(1, MyApp_Defer_1_Template, 1, 1);
+    $r3$.ɵɵtemplate(2, MyApp_DeferPlaceholder_2_Template, 3, 0);
+    $r3$.ɵɵdefer(0, 1, null, null, 2);
```

Comment:
As discussed offline, it'd be great to have indices of those instruction go incrementally.
```suggestion
    $r3$.ɵɵtemplate(0, MyApp_Defer_1_Template, 1, 1);
    $r3$.ɵɵtemplate(1, MyApp_DeferPlaceholder_2_Template, 3, 0);
    $r3$.ɵɵdefer(2, 1, null, null, 2);
```